docs/documentation/getting-started-intellij.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Getting Started with Saros/I
+---
+
+This is the first alpha release of Saros/I, so expect it to be a bit rough around the edges.
+There are still some [restrictions](#restrictions) that apply to the usage and some basic [features are still missing](#missing-features).
+This release is not compatible with any other Saros plugin/version (like Saros/E).
+
+The GUI used in Saros/I is only a placeholder providing the minimal, necessary functionality.
+We are currently working on a replacement in the form of an HTML GUI that can be used across all Saros versions.
+
+## Disclaimer
+
+Saros/I does not include sub-modules when sharing a module (see [Module Restrictions](#module-restrictions)).
+As a consequence, such sub-modules might not be present for all session participants.
+If a participant deletes a shared directory that contains a sub-module in the local setup of another participant, this sub-module will be deleted without any notice.
+
+## Features
```

Comment:
We have the separate page "Features" do you think it is useful to split this page (probably in the future)?

Reply:
Maybe, but I am not sure if this is really sensible. The current page states the complete Saros feature set that Saros/I is working towards. If we see in the process that the feature set is not completely transferable to Saros/I, we can have this discussion again.

The only issue in the meantime is that users could look at the feature list and then complain that something is not available in Saros/I. But this is mentioned in the release notes and the getting started section for Saros/I.